### PR TITLE
PT-574 Integrate latest change from 1.27.0

### PR DESCRIPTION
**Pull request by yvon-dblg**
Point of interests for a review (an rc4 will be needed I think):
- index.ejs: The zendesk script from tidepool have change. I leave it commented for now.
- server.js: All the URL.
- Even if there is env var defined in yourloops docker, they are not really used with the new Dockerfile from Tidepool. Should we use our old Dockerfile ?
- Translation check/update ?

Notes:
- @franck-fourel Travis script changed back to Ubuntu 14.04 Trusty  ;-)

### CODE REVIEW

File: app/components/browserwarning/browserwarning.js [hunk 1]
```diff
@@ -62,21 +75,35 @@ export default translate()(class BrowserWarning extends Component {
...
           {downloadCopy}
+          <div className="browser-warning-mobile">
+            <div className="browser-warning-mobile-message">
+              {t('Download Tidepool Mobile for iOS or Android to add notes and see your data on the go:')}
+            </div>
+            <div className="browser-warning-mobile-appstore-container">
+              <img alt='Download on the App Store' src={appstoreImageUrl} className="appstore-badge" onClick={handleClickiOS}/>
+              <img alt='Get it on Google Play' src={playstoreImageUrl} className="playstore-badge" onClick={handleClickAndroid}/>
+            </div>
+          </div>
```
[the remark this answers is not in the document]
Comment (yvon-dblg) [reply app/components/browserwarning/browserwarning.js]: For the translation I was aware (it wasn't updated, waiting other PR to be merged before).

File: app/pages/login/login.js [hunk 2]
```diff
@@ -152,7 +156,7 @@ export let Login = translate()(React.createClass({
   renderBrowserWarning: function() {
     const { t } = this.props;
 
-    if (!utils.isChrome()) {
+    if (!utils.isAcceptedBrowser()) {
```
Comment (CedricPerotto) [inline app/pages/login/login.js]: Why changing from Chrome to accepted browsers (FF, Edge), tests are only made only on Chrome by Tidepool and we know that some glitches can appear on FF and edge... the reason why we display this message for all non chrome users.
```suggestion
    if (!utils.isChrome()) {
```

File: app/core/utils.js [hunk 3]
```diff
@@ -67,23 +67,21 @@ utils.getIn = (obj, props, notFound) => {
...
 utils.isAcceptedBrowser = () => {
   var userAgent = navigator.userAgent.toLowerCase();
-  return (userAgent.indexOf('chrome') > -1 
-    || userAgent.indexOf('firefox') > -1 
-    || userAgent.indexOf('crios') > -1 
+  return (userAgent.indexOf('chrome') > -1
+    || userAgent.indexOf('firefox') > -1
+    || userAgent.indexOf('crios') > -1
     || userAgent.indexOf('edge') > -1);
 };
```
Comment (CedricPerotto) [inline app/core/utils.js]: See my other comment this function is useful.
```suggestion

utils.isChrome = () => {	
  var userAgent = navigator.userAgent.toLowerCase();	
  return (userAgent.indexOf('chrome') > -1 	
    || userAgent.indexOf('crios') > -1);	
};

```

File: locales/fr/translation.json [hunk 4]
```diff
@@ -183,7 +182,7 @@
   "Continue": "Continuer",
   "Copied!": "Copié !",
   "Copy as text": "Copier comme texte",
-  "Copy link": "Copier le lien",
+  "Copy this page\\’s URL": "Copier l\\'adresse de cette page",
```
Comment (CedricPerotto) [inline locales/fr/translation.json]: This key is not translated on the browser warning page.

**Comment by yvon-dblg** [comment]
> 
> 
> Missing translation key is now displayed

Can you be more specific ?

**Comment by CedricPerotto** [comment]
> > Missing translation key is now displayed
> 
> Can you be more specific ?

Just approving your last commit the the "Copy this page’s URL" translation key is now translated on the browser warning page. 

This PR is ready for merge.

**Comment by yvon-dblg** [comment]
See clean PR https://github.com/mdblp/blip/pull/122

**Pull request closed without merging**